hvplot/converter.py

```diff
@@ -498,6 +498,7 @@ def __init__(self, data, x, y, kind=None, by=None, use_index=True,
             plot_opts['global_extent'] = global_extent
         if projection:
             plot_opts['projection'] = process_crs(projection)
+        title = title if title else getattr(self, '_title', None)
```

Comment:
I don't know if numbers are allowed as plot titles, but if they are, seems like this logic will fail if someone passes 0 as the title (e.g. if they wanted to number a bunch of plots).  So maybe:

```suggestion
        title = title if title is not None else getattr(self, '_title', None)
```

That way people can also pass the empty string '' if they want to suppress the auto-generated title.